### DISCUSSION TITLE
12-hour format

### DISCUSSION THREAD

**Discussion by Pilloo** [polybar/polybar#2762] [Q&A]
Hi! Today I was configuring my bar and everything works fine, but when I was configuring the clock with the property "time = %I:%M %p" I noticed that the am/pm was not displayed, when in theory it should be displayed. How can I fix this? I read in an issue that this is the correct method, but it still doesn't work correctly. 

Screenshot of the bar:
![07-02-2022_21:46:46](https://user-images.githubusercontent.com/57467120/177023720-939ea9d2-1d1e-49f2-aafb-f1c72bd17393.png)

Configuration of the "date" module:
![07-02-2022_21:48:55](https://user-images.githubusercontent.com/57467120/177023775-e257c6b1-477c-41d3-8afd-52ce944ec190.png)

**Comment by patrick96** [comment]
`time = %I:%M %p` works for me, though whether `%p` shows anything seems to be dependent on your locale. I am using `en_US.UTF-8` as my locale.
